.github/workflows/publish_website.yml

```diff
@@ -34,8 +34,17 @@ jobs:
         uses: actions/setup-python@7f80679172b057fc5e90d70d197929d454754a5a # v2
         with:
           python-version: 3.x
-      - name: Installing Material for MkDocs
-        run: pip install mkdocs-material
+          cache: 'pip'
+      - name: Installing dependencies
+        run: |
+          pip install \
+            mkdocs-material \
+            requests
+      - name: Generating release index
+        run: |
+          build_tools/scripts/generate_release_index.py \
+            --repo="${GITHUB_REPOSITORY}" \
+            --output=docs/website/docs/pip-release-links.html
```

Comment:
Hmmm..

https://github.com/iree-org/iree/blob/7d53e568109d19ca00427e399a4c95f8191b75fa/.github/workflows/publish_website.yml#L8-L14

Now we want this to run on a schedule / after https://github.com/iree-org/iree/blob/main/.github/workflows/validate_and_publish_release.yml / promoting a release to stable too?

Reply:
I can trigger a build with workflow_dispatch manually for now, but then we should figure out what triggering makes the most sense.